ui/src/shared/components/InputClickToEdit.tsx

```diff
@@ -117,7 +117,7 @@ class InputClickToEdit extends PureComponent<Props, State> {
         </div>
       </div>
     ) : (
-      <div className={wrapperClass}>
+      <div className={wrapperClass} data-test="rename-org">
```

Comment:
"rename-org" id is not used in tests, is it?

Reply:
No it's not, should I remove it?

Reply:
Yes, pls and yes, a shared component must not rely upon instance-specific ID